### PR TITLE
[Coverity] Fix resource leak

### PR DESCRIPTION
**Pull request by gichan-jang**
Fix resource leak reported by coverity

### CODE REVIEW

**Review by songgot — APPROVED** [review]
LGTM

**Review by niley7464 — APPROVED** [review]
LGTM!

File: c/src/ml-api-remote-service.c [hunk 1]
```diff
@@ -385,6 +390,10 @@ _mlrs_process_remote_service (nns_edge_data_h data_h)
           "Service num: %d", service_type);
       break;
   }
+
+done:
+  g_free (service_key);
+  g_free (service_str);
```
Comment (wooksong) [inline c/src/ml-api-remote-service.c:396]: IMHO, ```service_str``` could be freed just after ```_mlrs_get_service_type()```.

Comment (gichan-jang) [reply c/src/ml-api-remote-service.c]: Thanks, I added g_autofree macro for `service_str` and `service_key`.

**Review by wooksong — APPROVED** [review]

**Review by again4you — APPROVED** [review]
Good Job!

**Pull request merged**